examples/chip/minimal.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+from os import uname
+from socket import gethostname
+from time import sleep
+import sys
+import cloud4rpi
+import chip
```

Comment:
maybe we should rename this one, because the file you refer to (raspberrypi/minimal.py) is platform-independent.